### PR TITLE
application update call implemented. Minor refactoring.

### PR DESCRIPTION
**Pull request by raja-sami-10p**

### CODE REVIEW

File: lib/facade/application.js [hunk 1]
```diff
@@ -85,11 +86,11 @@ function updateApp(reqData, appId) {
...
       }
     ])
-    .then((res) => {
-      console.log("****************");
-      console.log(res);
-      return "promise accepted";
-    })
+    .then(helper.checkBooleanPair.bind(this, payload.isActive, payload.isDeleted))
+    .then(appRepo.update.bind(this, payload))
+    .then(function (response) {
```
Comment (faizulhaque-tp) [inline lib/facade/application.js:91]: seems like unnecessary function closure

File: lib/facade/user.js [hunk 2]
```diff
@@ -18,8 +17,10 @@ function logOutUser(userId, token) {
    * Remove the token of such user. */
   return tokenLookUp.findOneAndRemove({userId: userId, accessToken: token})
     .then(function (object) {
-      if (!object) {
-        throw new errorHandler.unAuthenticatedError('User not found');
+      if (object) {
+        return ({});
```
Comment (faizulhaque-tp) [inline lib/facade/user.js:21]: reason for ({})?

File: lib/facade/user.js [hunk 3]
```diff
@@ -33,7 +34,7 @@ getOrCreateToken.prototype.retrieveToken = Promise.coroutine(function* (username
 
   /**Authenticate user trying to login.*/
   let authenticatedUser = yield userAccount.Authenticate(username, password);
-  if (!authenticatedUser) throw new errorHandler.unAuthenticatedError('User not registered.');
+  if (!authenticatedUser) throw new errorHandler.unAuthenticatedError(errorHandler.messages.USER_NOT_REGISTERED);
```
Comment (faizulhaque-tp) [inline lib/facade/user.js:37]: nice :+1:

File: lib/facade/user.js (hunk 2 above)
Comment (raja-sami-10p) [inline lib/facade/user.js:21]: empty return was not recommended by Webstrom editor

**Pull request merged**